### PR TITLE
Laetitia week 2

### PR DESCRIPTION
**Pull request by supertata**
# Your Details

Your Name:
Your City:
Your Slack Name:

# Homework Details

Module:
Week:

### CODE REVIEW

File: week-2/Homework/mandatory/4-sorting-algorithm.js [hunk 1]
```diff
@@ -20,59 +20,77 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
...
+let arr = [1, 45, 43, "Hello", "Goodbye"]
+let newArr = []
 function sortAges(arr) {
-  //create function here
+  
+  for (let i = 0; i < arr.length; i++) {
+    
+    if (typeof arr[i] === 'number') {
+      newArr.push(arr[i])
+    }
```
Comment (zoltan-gal) [inline week-2/Homework/mandatory/4-sorting-algorithm.js:32]: So far, it looks good.
If you wouldn't like to write your own sorting algorithm then you can go for the simple way by using the built-in `sort()` method on array.

File: week-2/Homework/mandatory/3-playing-computer.js [hunk 2]
```diff
@@ -12,37 +12,60 @@
...
+  1. This program throws an error. Why? (If you can't find it, try executing it). 
+     // b is not defined - replace x by b. 
+
   2. Remove the line that throws the error.
+     
   3. What is printed to the console?
-  4. How many times is "f1" called?
+  2, 6, 0, 2, 4, 6, 9, 13.    // @ mentor can you please help me understand why the output is 2, 6, 0, 4, 9, 6, 13, 8? 
+                              // you can see my reasoning in my notes below. Where am I wrong? 
+
```
Comment (zoltan-gal) [inline week-2/Homework/mandatory/3-playing-computer.js:23]: It should be: 2 6 4 9 6 13 8
You have an extra 0 after 6.

Comment (zoltan-gal) [reply week-2/Homework/mandatory/3-playing-computer.js:23]: Please get in touch with your mentor and ask for explaining it a live coding call, it'd be easier and faster to understand if you see it in live.

**Comment by zoltan-gal** [comment]
Nice job @supertata! I'd recommend a revise how to manipulate array elements and how to use expressions.
In addition, please pay more attention to keep your code neat and tidy, neat indentation might be a bit tedious but helps a lot for readers to reason about the code. 

Here is a short document about some convention Javascript developers follows. I recommend to study it: https://www.w3schools.com/js/js_conventions.asp

File: week-2/Homework/mandatory/3-playing-computer.js (hunk 2 above)
Comment (supertata) [reply week-2/Homework/mandatory/3-playing-computer.js:23]: Thanks for the feeback @zoltan-gal. I'll work on the exercises again after I'm done with this week's homework.

**Pull request closed without merging**